ultraleap/TouchFree
# TF-300: TouchFree running on BrightSign

- [ ] Test plan updated (by PR creator) 
- [x] Developer Testing: Key item tests 
- [ ] Developer Testing: Execution of the updated test plan (changed items for this work only) 
- [ ] Product Owner Review (Optional) 
- [ ] Update Release Notes on the JIRA ticket for harvest at release time

The contents of the ARMLinux build job artefact should contain everything required to run TouchFree. To actually run TouchFree on a BrightSign box:

1. Copy the contents of the aforementioned artefact onto a microsd card
2. Place that card into a development BrightSign box with `telnet` enabled
3. Connect via `telnet` to that BrightSign box
4. Copy the contents of the card to `/tmp/`
5. Create the following symlinks:
  5a. `libatomic.so.1.2.0 -> libatomic.so.1`
  5b. `libLeapC.so.5 -> libLeapC.so`
  5c. `libstdc++.so.6.0.28 -> libstdc++.so.6`
6. Run the service using this command: `LD_LIBRARY_PATH=. ./TouchFree\ Service`

The above will be covered for production environments as part of the work in TF-302
